etc/json/other_modules.json

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
why?

Reply:
This is a list of non-blocking modules. Without using this module file, autosploit is not usable against multiple host since most fuzzers are hanging for an indefinite amount of time.

Reply:
So it's basically the same as `default_modules.json` without fuzzers correct?

Reply:
Yes

Reply:
So why wouldn't we just fix the fuzzers instead of creating a duplicate file?

Reply:
It's a dev branch; I'm just putting a file I'm able to work with, without deleting stuff some other might rely on. 
Feel free to modify the default_modules.json and drop the other_modules.json file.

Reply:
yeah just delete this and I'll merge the vagrant